native/src/components/Header.tsx

```diff
@@ -159,6 +159,8 @@ const Header = (props: PropsType): ReactElement => {
       <Item title={title} accessibilityLabel={accessibilityLabel} iconName={iconName} show={show} onPress={onPress} />
     )
   }
+  // render placeholder while loading to spare space for icon
+  const renderIconPlaceholder = (title: string) => <Item style={{ opacity: 0 }} title={title} iconName='search' />
```

Comment:
Why don't you just add a property visible to the renderItem function above and set opacity to zero there if true?

Reply:
yes i had this before but regarding to readability i found this approach nicer.
But i can change this

Reply:
Lets wait for other opinions here :)

Reply:
And i would have to adjust all the renderItem Fkt-Calls just for these 2 Cases and add a boolean value

Reply:
You could use true as default, then you only have to adjust those two calls

Reply:
I think I prefer it in one function, if you can make it is clear that renderItem can return a placeholder e.g. by passing the availabilityFlags..

Reply:
inschallah :)